### PR TITLE
Resolve #2301

### LINKED ISSUE #2301: Windows Server 2008 R2 not supported?
[Enter feedback here]
As per the text in this document the Get-DnsServer cmdlet is supported in Windows Server 2008 R2. 
I cannot see that it's possible. I've tested with a fully patched Windows Server 2008 R2 (without ESU) with Powershell 5.1 and the command is unrecognized.

Thanks,
/Kenny

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 6df784d1-fcec-ba75-56c1-17ff67e914b6
* Version Independent ID: 1928b38e-cd31-45e4-a927-ee17e477fb61
* Content: [Get-DnsServer (DnsServer)](https://docs.microsoft.com/en-us/powershell/module/dnsserver/get-dnsserver?view=win10-ps)
* Content Source: [docset/windows/dnsserver/Get-DnsServer.md](https://github.com/MicrosoftDocs/windows-powershell-docs/blob/master/docset/windows/dnsserver/Get-DnsServer.md)
* Product: **w10**
* Technology: ****
* GitHub Login: @Kateyanne
* Microsoft Alias: **v-kaunu**

### PR DESCRIPTION
**Pull request by velkovb**
Remove Server 2008 R2
https://github.com/MicrosoftDocs/windows-powershell-docs/issues/2301

### CODE REVIEW

File: docset/winserver2012-ps/dnsserver/Get-DnsServer.md [hunk 1]
```diff
@@ -22,23 +22,22 @@ Get-DnsServer [-AsJob] [-CimSession <CimSession[]>] [-ComputerName <String>] [-T
 
 ## DESCRIPTION
 The **Get-DnsServer** cmdlet retrieves a Domain Name System (DNS) server configuration.
-The DNS server must be running Windows Server® 2008 R2 operating system or above.
 
 You can pipe the output of the **Get-DnsServer** cmdlet to the **Export-Clixml** cmdlet, which generates an XML file of the configuration.
 You can use the XML file to back up or transfer DNS settings between computers.
-For more information about **Export-Clixml**, see Using the Export-Clixml Cmdlethttp://technet.microsoft.com/en-us/library/ee176824.aspx.
+For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://technet.microsoft.com/en-us/library/ee176824.aspx).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/dnsserver/Get-DnsServer.md:28]:
```suggestion
For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://technet.microsoft.com/library/ee176824.aspx).
```

File: docset/winserver2012r2-ps/dnsserver/Get-DnsServer.md [hunk 2]
```diff
@@ -30,23 +30,22 @@ Get-DnsServer [-ComputerName <String>] [-CimSession <CimSession[]>] [-ThrottleLi
 
 ## DESCRIPTION
 The **Get-DnsServer** cmdlet retrieves a Domain Name System (DNS) server configuration.
-The DNS server must be running Windows Server® 2008 R2 operating system or above.
 
 You can pipe the output of the **Get-DnsServer** cmdlet to the **Export-Clixml** cmdlet, which generates an XML file of the configuration.
 You can use the XML file to back up or transfer DNS settings between computers.
-For more information about **Export-Clixml**, see Using the Export-Clixml Cmdlethttp://technet.microsoft.com/en-us/library/ee176824.aspx.
+For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://technet.microsoft.com/en-us/library/ee176824.aspx).
```
Comment (JohanFreelancer9) [inline docset/winserver2012r2-ps/dnsserver/Get-DnsServer.md:36]:
```suggestion
For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://technet.microsoft.com/library/ee176824.aspx).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by get-itips — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2012-ps/dnsserver/Get-DnsServer.md [hunk 3]
```diff
@@ -22,23 +22,22 @@ Get-DnsServer [-AsJob] [-CimSession <CimSession[]>] [-ComputerName <String>] [-T
 
 ## DESCRIPTION
 The **Get-DnsServer** cmdlet retrieves a Domain Name System (DNS) server configuration.
-The DNS server must be running Windows Server® 2008 R2 operating system or above.
 
 You can pipe the output of the **Get-DnsServer** cmdlet to the **Export-Clixml** cmdlet, which generates an XML file of the configuration.
 You can use the XML file to back up or transfer DNS settings between computers.
-For more information about **Export-Clixml**, see Using the Export-Clixml Cmdlethttp://technet.microsoft.com/en-us/library/ee176824.aspx.
+For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://technet.microsoft.com/library/ee176824.aspx).
```
Comment (tseward) [inline docset/winserver2012-ps/dnsserver/Get-DnsServer.md:28]: Please update the URL to https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/export-clixml

File: docset/winserver2012-ps/dnsserver/Get-DnsServer.md [hunk 4]
```diff
@@ -22,23 +22,22 @@ Get-DnsServer [-AsJob] [-CimSession <CimSession[]>] [-ComputerName <String>] [-T
 
 ## DESCRIPTION
 The **Get-DnsServer** cmdlet retrieves a Domain Name System (DNS) server configuration.
-The DNS server must be running Windows Server® 2008 R2 operating system or above.
 
 You can pipe the output of the **Get-DnsServer** cmdlet to the **Export-Clixml** cmdlet, which generates an XML file of the configuration.
 You can use the XML file to back up or transfer DNS settings between computers.
-For more information about **Export-Clixml**, see Using the Export-Clixml Cmdlethttp://technet.microsoft.com/en-us/library/ee176824.aspx.
+For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/export-clixml).
```
Comment (JohanFreelancer9) [inline docset/winserver2012-ps/dnsserver/Get-DnsServer.md:28]:
```suggestion
For more information about **Export-Clixml**, see [Using the Export-Clixml cmdlet](https://docs.microsoft.com/powershell/module/microsoft.powershell.utility/export-clixml).
```

**Review by tseward — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Review by tseward — APPROVED** [review]

**Pull request merged**